### DISCUSSION TITLE
Touch Bar support

### DISCUSSION THREAD

**Discussion by Zakstensrud4590** [ciderapp/Cider#1188] [Feature Request]
I would like to see Touch Bar support in Cider similar to the stock Apple Music app which would allow you to scrub though and control both play/pause and rewind/forward.

**Comment by cryptofyre** [comment]
We're currently looking at this, but unfortunately we don't have any good way of testing it as my MacBook is the 2021 Pro without the touchbar lol.

**Reply by coredev-uk** [reply]
I may be able to set this up, either with touchbar emulator, or with a MacBook Pro. I’ll get back to you when I begin.

**Reply by Zakstensrud4590** [reply]
Sounds good!

**Reply by Zakstensrud4590** [reply]
If you want it to be tested on a real touch bar before you release it I can test it on my M1 MacBook Pro.
